code/05-creating-histograms/ColorHistogramMask.py

```diff
@@ -3,28 +3,29 @@
  *
  * Usage: python ColorHistogramMask.py <filename>
 '''
-import cv2
-import numpy as np
 import sys
+import skimage.draw
+import skimage.io
+import skimage.viewer
+import numpy as np
 from matplotlib import pyplot as plt
 
 # read original image, in full color, based on command
 # line argument
-image = cv2.imread(filename = sys.argv[1])
+image = skimage.io.imread(fname = sys.argv[1])
 
-# display the original image 
-cv2.namedWindow(winname = "Original Image", flags = cv2.WINDOW_NORMAL)
-cv2.imshow(winname = "Original Image", mat = image)
-cv2.waitKey(delay = 0)
+# display the original image
+viewer = skimage.viewer.ImageViewer(image)
+viewer.show()
 
 # create a circular mask to select the 7th well in the first row
 # WRITE YOUR CODE HERE
 
-# use cv2.bitwise_and() to apply the mask to img, and save the 
+# use np.logical_and() to apply the mask to img, and save the
```

Comment:
i should probably revisit the masking episode. I think I inverted everything there so that I could use indexing to apply masks. So I used `ones` for the initial mask, then set everything i wanted to keep to `False`, indexed with the mask to set stuff outside the roi to 0...